### ISSUE TITLE
Renamed a package and now Update Universe action always fails

### ISSUE THREAD

**Issue by elipousson** [r-universe-org/help#490]
Last fall, I renamed this small R package I wrote from settoken to standaloner: https://github.com/elipousson/standaloner

Initially, I forgot that I had listed settoken as a package in the configuration for my R Universe page: https://github.com/elipousson/elipousson.r-universe.dev

I caught the issue and removed settoken/standaloner from the packages.json file about 8 months ago. The package has no current dependencies from any other of my packages (although I think it did in the past)

However, R Universe is still unable to update due to an error related to this renamed package: https://github.com/r-universe/elipousson/actions

I'm unsure how to resolve – any tips or advice on how to fix this are much appreciated!

**Comment by elipousson** [comment]
I think I might have figured this out - I still had the old name set under Remotes in the DESCRIPTION of one of my other packages. Hoping this change may resolve the issue and will close this issue if that turns out to be the case.

**Comment by jeroen** [comment]
If you look at the CI ❌  in your repo https://github.com/elipousson/elipousson.r-universe.dev you will also see the error

**Comment by elipousson** [comment]
Got it. I fixed this by:

- Removing the Remotes for settoken in the bingmapr package
- Removing the duplicate entry for maplayer package in my packages.json

I think I also had to update the version for any package that previously depended on settoken (where I hadn't incremented the version updated since removing the dependency).

I should probably take this as a nudge to get better about versioning my packages any time I touch the dependencies. Thanks for your help in figuring this out!
